store/postgres/migrations/2020-04-10-111111_add_deployment_errors/up.sql

```diff
@@ -0,0 +1,54 @@
+-- create subgraph_error
+create table subgraphs."subgraph_error" (
+        "id"                 text not null,
+        "subgraph_id"        text not null,
+        "message"            text not null,
+        "block_number"       numeric,
```

Comment:
Can this ever be different from `lower(block_range)`?

Reply:
The operation to create this entity is currently unversioned. If you think it is viable I could make this a versioned entity, which would make sense since non-fatal errors should be cleared on a revert.

It will be a bit inconvenient to lose the ability to sort on `block_number` in a graphql query, but I can just write a lower-level query to order the `nonFatalErrros`.

Reply:
Yes, I agree; exposing this in GraphQL is valuable, and the redundant data field is the easiest way to get there. Could it be an `i32`?

Reply:
In our metadata we use `BigInt` for block numbers which corresponds to `numeric` in sql.

Reply:
Since this can be made versioned in a future PR without changing the schema, I'm thinking of not doing any change here for now.